python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -798,16 +894,45 @@ def _addSchemaKeys(self, schema):
                                               doc="Flux measurement from scarlet")
         self.scarletLogLKey = schema.addField("deblend_logL", type=np.float32,
                                               doc="Final logL, used to identify regressions in scarlet.")
+        self.edgePixelsKey = schema.addField('deblend_edgePixels', type='Flag',
+                                             doc='Source had flux on the edge of the parent footprint')
         self.scarletSpectrumInitKey = schema.addField("deblend_spectrumInitFlag", type='Flag',
                                                       doc="True when scarlet initializes sources "
                                                           "in the blend with a more accurate spectrum. "
                                                           "The algorithm uses a lot of memory, "
                                                           "so large dense blends will use "
                                                           "a less accurate initialization.")
-
-        # self.log.trace('Added keys to schema: %s', ", ".join(str(x) for x in
-        #               (self.nChildKey, self.tooManyPeaksKey, self.tooBigKey))
-        #               )
+        self.nComponentsKey = schema.addField("deblend_nComponents", type=np.int32,
+                                              doc="Number of components in a ScarletLiteSource. "
+                                                  "If `config.version != 'lite'`then "
+                                                  "this column is set to zero.")
+        self.psfKey = schema.addField('deblend_deblendedAsPsf', type='Flag',
+                                      doc='Deblender thought this source looked like a PSF')
+        # Blendedness/classification metrics
+        self.maxOverlapKey = schema.addField("deblend_maxOverlap", type=np.float32,
+                                             doc="Maximum overlap with all of the other neighbors flux "
+                                                 "combined."
+                                                 "This is useful as a metric for determining how blended a "
```

Comment:
Really, I thought that we were? I've always done that in my field docs.

Reply:
Where would we put the usage suggestion? Is there guidance on this in the dev guide?

Reply:
I don't think so, and said as much on Jira. I guess I'm fine with a brief explanatory comment like "values close to the noise level indicate minimal blending". I assume most of the columns that end up in `objectTable`s are defined in LPMs or similar docs. If usage suggestions are based on applications to specific datasets like test-med-1/RC2, those should probably go in a DMTN.

Reply:
I agree that we do need deblender docs and it's on me for not having a good DMTN or page(s) in the pipeline docs to discuss how meas_extensions_scarlet works and how its outputs should be used. I've just put off making them because I want things to feel more finished. I know that's a bar that can easily continue to keep moving but I feel like the current blockers are a fair place to start. See DM-30535, which I just updated with an appropriate set of blockers.